src/renderer/components/notification-comp.tsx

```diff
@@ -210,7 +285,35 @@ export default class NotificationComp extends React.Component<{}, IState> {
      * @param id {number}
      */
     private onMouseLeave(id: number): void {
-        ipcRenderer.send('notification-mouseleave', id);
+        const { isInputHidden } = this.state;
+        ipcRenderer.send('notification-mouseleave', id, isInputHidden);
+    }
+
+    /**
+     * Insets a thumbs up emoji
+     * @private
+     */
+    private onThumbsUp(): void {
+        if (this.input.current) {
+            const input = this.input.current.value;
+            this.input.current.value = input + '👍';
```

Comment:
Hilarious that you can have emoji inside code I think

Reply:
@KiranNiranjan Does this get converted to unicode across OSes? Hope this is not specific to macOS or Windows.

Reply:
@VishwasShashidhar While sending the message it will be sent as `:thumbsup:` which Client app supports